laterpay/views/backend/pricing.php

```diff
@@ -31,11 +31,64 @@ class="lp_plugin-mode-indicator"
         $selected_option = ( int ) get_option( 'laterpay_post_price_behaviour', 2 );
         $hasCategories   = ( count( $laterpay['categories_with_defined_price'] ) > 0 );
         $isGlobalTypeOne = ( 1 === $selected_option );
+
+        $hidden_post_types  = $laterpay['hidden_post_types'];
+        $all_post_types     = $laterpay['all_post_types'];
+        $enabled_post_types = $laterpay['enabled_post_types'];
         ?>
     </div>
 
     <div class="lp_pagewrap">
 
+        <div class="lp_enabled_post_types">
+            <h2><?php esc_html_e( 'Default Pricing', 'laterpay' ); ?></h2>
+            <p>
+                <?php
+                printf(
+                    '%1$s <br/> %2$s',
+                    esc_html__( 'All WordPress Posts, Pages and Media will display the pricing options below unless this is overridden by setting an alternate', 'laterpay' ),
+                    esc_html__( 'Individual Article Price on the specific page or by a rule established on the Custom Pricing Tab.', 'laterpay' )
+                );
+                ?>
+            </p>
+
+            <form id="lp_js_globalEnabledPostTypesForm" method="post" action="">
+                <input type="hidden" name="form"    value="update_enabled_post_types">
+                <input type="hidden" name="action"  value="laterpay_enabled_post_types">
+                <?php if ( function_exists( 'wp_nonce_field' ) ) { wp_nonce_field( 'laterpay_form' ); } ?>
```

Comment:
Any reason for function exists check?

Reply:
I checked existing forms for nonce implementation. The check is added almost everywhere. Can remove it, no issue.